src/workerd/io/actor.h

```diff
@@ -0,0 +1,118 @@
+#include "worker.h"
```

Comment:
I'd recommend against having a header with implementation in a source file with a different stem. Many build systems assume source and header files with the same stem are part of the same object. Note that c++20 modules are somewhat like source files that can be preprocessed like headers. Assuming this commit is just code movement, I'd recommend doing a two commit process where first you separate out the class in one commit and then you move it to a new file in another commit.

Reply:
Joy, this is tightly coupled. I would really like to break this coupling. It looks like we could take make a struct like `PersistentStorage` that has a `rpc::ActorStorage::Stage::Client` and a `ActorCache::SharedLru &` to take in the ctor. For `buildClassInstance()`, perhaps that could stay in `worker.h` since we only use it in source files?